concordium-sdk/src/main/java/com/concordium/sdk/requests/getinstanceinfo/ContractAddress.java

```diff
@@ -0,0 +1,20 @@
+package com.concordium.sdk.requests.getinstanceinfo;
+
+import com.concordium.sdk.serializing.JsonMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Smart Contract Address.
+ */
+@Builder
+@Getter
+public class ContractAddress {
```

Comment:
There's already a `ContractAddress` defined in the SDK. I think you can use that instead.